### ISSUE TITLE
Add wireless adapter support

### ISSUE THREAD

**Issue by tolik518** [evanbowman/skyland-beta#17]
As there is already a multiplayer mode and the link cable is very unreliable I'd suggest to
add support for the wireless adapter.

There is already [a library](https://github.com/rodri042/gba-link-connection/blob/master/lib/LinkWireless.h) that supports the wireless adapter, which is used by [piugba](https://github.com/rodri042/piugba) for example.

I couldn't test is though - I do have 2 wireless adapters, but only one GBA

**Comment by evanbowman** [comment]
Yeah! I'd like to play around with wireless adapters, but unfortunately I don't have any. The link cable isn't really unreliable in my experience though. There's no risk of data loss with the cable unless it comes unplugged.

**Comment by tolik518** [comment]
If you'd be in Europe I'd be happy to borrow you mine, but I guess it would be cheaper to buy new ones than paying shipping cost the transatlantic shipping.

>  The link cable isn't really unreliable in my experience though.

I guess I am biased then from cables with loose contacts.

**Comment by Doronimmo** [comment]
This could be tested with retroarch gpsp with netplay

**Comment by Doronimmo** [comment]
https://github.com/afska/gba-link-connection
This library could be useful

**Comment by evanbowman** [comment]
> https://github.com/afska/gba-link-connection This library could be useful

The library is useful as an example of how the wireless protocol works, but gba-link-connection also has some unfortunate design limitations that would make it impossible to use in this project. I may refer to gba-link-connection sometime in the future as an example of wireless multiplayer code.
